### DISCUSSION TITLE
Feature Request:  Fragment Open Telemetry Documentation on homepage by "User".

### DISCUSSION THREAD

**Discussion by jsuereth** [open-telemetry/opentelemetry.io#4304] [Content Refactoring]
I'd really love if the Open Telemetry homepage had the following content:

- A broad overview of what Open Telemetry is.
- Three sets of links for users to continue reading based on what type of user they may be (for now I'm thinking `users`, `integrators` and `maintainers`).

Then there'd be a new homepage with geared information and links.

## For Users
This would be a section for end-users on how to leverage OT for custom spans/metrics/logging and set-up configure an OT system.  It'd be great if basic architectural concerns and links for "how to" on language-level integrations.  Effectively this would have:

- API Documentation
- Collector Configuration Documentation
- Example Architectures
- Example services/setups
- Registry of integrations

TL;DR: This is "How/why should I use OT in my app deployments"

## For Integrators
This would be a homepage geared towards other projects trying to use OT APIs to provide spans/metrics/logs for their users.   This would include:

- SDK Documentation
- "Best practices" on exporting spans, metrics, etc. for frameworks
- Links to example exporters
- Links to example framework-integrations (gRPC e.g.)

TL;DR: This would be "How/why should I use OT in my library/framework/service"

## For Maintainers
This would be a homepage geared that outlines how to get started contributing and the breadth-scope of the project.

- Links to github repositories
- Links to "how to contribute" and process
- Links to specification.

TL;DR: This would be "How to I add features or fix bugs"

## Example website splits of this sort:

- [Apache Spark](https://spark.apache.org/) - see the bottom calls out Community, Contributors, Getting Started
- [Ubuntu](https://ubuntu.com/) - An extreme example where they call out a myriad of use cases, example users and links
- [Play Framework](https://www.playframework.com/) - Get Started(New User), Read the Docs(Current User), Get Involved(Maintainer)
- [Micronaut](https://micronaut.io/) - Three points to jump off, more info scrolling down.

I'm happy to discuss further ideas and happy to help provide content to address the three use cases.

**Comment by austinlparker** [comment]
I think the user/integrator stuff would be great for the new docs section, definitely. I'm going to try to get the new theme out ASAP so we can build content for it.

**Comment by chalin** [comment]
FYI, I just linked this to #770, which will be used to track progress on the IA rework.

**Comment by svrnm** [comment]
We have something like that nowadays at https://opentelemetry.io/docs/getting-started/, but I am wondering if this is 100% the same, as this issue also talks about having integrators and maintainers listed (where Dev & Ops are "Users" in this definition). I think we can and should iterate on that.
